037-kafka-canary-roadmap.md

```diff
@@ -0,0 +1,127 @@
+# Kafka Canary roadmap
+
+This proposal describes a possible future set of improvements and features for the [Strimzi Canary](https://github.com/strimzi/strimzi-canary) tool, which is used to check that Kafka clusters are operating correctly.  
+
+## Current situation
+
+The Canary project [README.md](https://github.com/strimzi/strimzi-canary/blob/main/README.md) explains the current feature set.
+
+The Canary was added to Strimzi under [proposal 13](013-kafka-canary.md).
+The implementation is in Golang and uses the [Sarama](https://github.com/Shopify/sarama/) Kafka client.
+
+## Motivation
+
+We believe the Canary component could deliver significant value for Strimzi users. For example, Canary could be used as a basis for an SLI on cluster availability and health.
```

Comment:
I think the Canary is absolutely failing in this aspect as the information it provides is absolutely not user friendly and requires a lot of expertise to understand. This statement is a bit confusing for me -> it is not clear if it implies any changes (in that case they need to be elaborated on) or just repeats what was said before.